USAGE.rst

```diff
@@ -4,6 +4,13 @@
 
    You need to have an :ref:`environment loaded <install-profile>` (``source $HOME/picongpu.profile``) that provides all :ref:`PIConGPU dependencies <install-dependencies>` to complete this chapter.
 
+.. warning::
+
+   PIConGPU source code is portable and can be compiled on all major operating systems.
+   However, helper tools like ``pic-create`` and ``pic-build`` described in this section rely on Linux utilities and thus are not expected to work on other platforms out-of-the-box.
```

Comment:
`Linux.utilities` should be `Linux utilities`

Reply:
I am not sure what do you mean, to me it seems to already be "Linux utilities". Could you elaborate?

Reply:
There was a dot in between the words. But looks like it is now fixed